docs/level-23.md

```diff
@@ -93,7 +92,9 @@ import TrashPushDischarge from '@site/image-generator/yml/level-23/trash-push-di
 
 <TrashPush />
 
-- The known-trash interpretation also holds if the team has to prevent the clue receiver to *Trash Push* a card that's not playable:
+- For example, if an *Unnecessary Trash Push Discharge* was performed by cluing a card in Cathy's hand, Bob should blind-play his *Third Finesse Position* (from the clue). Note that, even though the focus of the card touched by the clue was known trash, Bob had to blind-play to tell Cathy that the card she was about to push is trash, so the 'unknown trash' rules apply. Therefore Cathy also performs a *Trash Push*, playing the card next to the card that she learned is trash.
+
```

Comment:
no newline here

Reply:
(I don't know if pressing the resolved button also alerted, so just to be sure:) Thanks, sorry about overlooking that, resolved.